android/features/login/src/main/java/io/newm/feature/login/screen/createaccount/CreateAccountScreenPresenter.kt

```diff
@@ -69,12 +70,14 @@ class CreateAccountScreenPresenter(
                             coroutineScope.launch {
                                 step = Step.Loading
                                 step = try {
-                                    recaptchaClientProvider.get().execute(RecaptchaAction.SIGNUP).onSuccess { token ->
-                                        signupUseCase.requestEmailConfirmationCode(
-                                            email = userEmail.text,
-                                            humanVerificationCode = token
-                                        )
-                                    }.onFailure {
+                                    recaptchaClientProvider.get()
+                                        .execute(RecaptchaAction.custom("auth_code"))
```

Comment:
cc: @martyu When calling the `requestEmailConfirmationCode` we need to ensure we pass in a custom action to recaptcha `auth_code`